### DISCUSSION TITLE
Reset Start Menu (Win10)

### DISCUSSION THREAD

**Discussion by Rayray1703** [Open-Shell/Open-Shell-Menu#1647] [Q&A]
i want my start menu to look like windows 10 again but i cant find out how to do it

**Comment by ge0rdi** [comment]
You can uninstall `Open-Shell` (or just its start menu component if you want to keep Classic Explorer).

Or you can configure what menu is opened by left-click (Win key) in `Open-Shell menu settings` -> `Controls`.
